core/src/test/scala/integration/kafka/api/CustomQuotaCallbackTest.scala

```diff
@@ -38,11 +40,9 @@ import java.util.Properties
 import java.util.concurrent.ConcurrentHashMap
 import java.util.concurrent.atomic.{AtomicBoolean, AtomicInteger}
 import java.{lang, util}
-import scala.collection.Seq
 import scala.collection.mutable.ArrayBuffer
 import scala.jdk.CollectionConverters._
 
-@Disabled("KAFKA-18213")
```

Comment:
nice to see this test gets fixed. Could you please close KAFKA-18213 as duplicate?

Reply:
I move it to L104, I didn't fix it

Reply:
I will take a look in this fail test